boards/DIM/Src/Io/Io_SevenSegDisplays.c

```diff
@@ -74,15 +65,14 @@ void Io_SevenSegDisplays_WriteCommands(void)
     // The 7-segment displays are daisy chained by shifting registers, so we
     // can't update them individually. Instead, we must update the 7-segment
     // displays all at once.
-    for (int display = 0; display < TEMP_NUM_DISPLAYS; display++)
+    for (int display = 0; display < 9; display++)
     {
-        // TODO: Update commands for 9 displays
-        const uint8_t display_data = formula_e_commands[TEMP_NUM_DISPLAYS - display - 1]; // invert order
+        const uint16_t display_data = commands[9 - display - 1]; // invert order
 
         for (int i = 0; i < SHIFT_REGISTER_SIZE; i++)
         {
             const bool bit_state = IS_BIT_SET(display_data, i);
-
+            // TODO correct pinouts
```

Comment:
is this still outstanding?

If this is to be pushed to another PR, use // TODO: syntax so it gets labelled as a todo item in Clion